### ISSUE TITLE
Revise ProductAccessibility instances/schema (openMINDS_controlledTerms)

### ISSUE THREAD

**Issue by Raphael-Gazzotti** [openMetadataInitiative/openMINDS_instances#260]
The current ProductAccessibility instances define access based on payment status (freeAccess and paidAccess). However, accessibility should be independent of whether a product is free or paid.

This would involve decoupling accessibility from payment status.
We may also introduce new access types to cover authentication-based restrictions (such as registeredAccess, authenticationAccess authorizedAccess).

Reference:
[ProductAccessibility Documentation](https://openminds-documentation.readthedocs.io/en/latest/instance_libraries/terminologies/productAccessibility.html)

**Comment by apdavison** [comment]
> accessibility should be independent of whether a product is free or paid

why?

**Comment by lzehl** [comment]
related to #132 

HERE MY SUGGESTION:

ProductAccessModes:

    virtual
    physical

ProductAccessTypes: (former ProductAccessibility)

    open access
    login-controlled access (only login required)
    consent-controlled access (login + consent required)
    payment-controlled access (payment required)
    admin-controlled access (admin grants access on demand)
    review-controlled access (review committee/process for access grant)
    mediated access (no direct access; only through admin or system)

AccessLocation:

    on-site
    off-site
    mediated on-site

**Comment by lzehl** [comment]
For software and services we may want to distinguish the code availability (stated below) from the accessibility of using the products functions (stated above):

```
open-source
closed-source
source-available
```

**Comment by lzehl** [comment]
Payment-controlled access can also be further split into:
```
single-payment controlled access
recurring-payment controlled access
```

**Comment by apdavison** [comment]
An idea for what a new Accessibility schema and associated controlled terms could look like, following on from discussion of axes in the developers' meeting yesterday:

```
Accessibility
    mode: AccessMode [virtual, physical]
    type: AccessType [open, authorization-required, mediated]
    payment: PaymentType [none, one-time, recurring]
    authorizationProcess: AuthorizationProcess [none, registration-only, admin-controlled, review-controlled, owner-controlled]
```

**Comment by lzehl** [comment]
feedback from @MBAbrams @archgogo 
- maybe use "on-site" instead of "physical"
- maybe use "committee-controlled" instead of "review-controlled"

**Comment by Raphael-Gazzotti** [comment]
From the Publication Status Ontology
Access type could be mapped to the members of [PublicationStatus](http://purl.org/spar/pso/PublicationStatus) i.e.:
[accepted for publication](https://sparontologies.github.io/pso/current/pso.html#d4e533)ni, [anonymized](https://sparontologies.github.io/pso/current/pso.html#d4e546)ni, [archived](https://sparontologies.github.io/pso/current/pso.html#d4e560)ni, [catalogued](https://sparontologies.github.io/pso/current/pso.html#d4e573)ni, [closed access](https://sparontologies.github.io/pso/current/pso.html#d4e586)ni, [confidential](https://sparontologies.github.io/pso/current/pso.html#d4e600)ni, [copy-edited](https://sparontologies.github.io/pso/current/pso.html#d4e613)ni, [corrected](https://sparontologies.github.io/pso/current/pso.html#d4e626)ni, [draft](https://sparontologies.github.io/pso/current/pso.html#d4e640)ni, [embargoed](https://sparontologies.github.io/pso/current/pso.html#d4e653)ni, [enhanced](https://sparontologies.github.io/pso/current/pso.html#d4e666)ni, [final draft](https://sparontologies.github.io/pso/current/pso.html#d4e680)ni, [gold open access](https://sparontologies.github.io/pso/current/pso.html#d4e693)ni, [gratis open access](https://sparontologies.github.io/pso/current/pso.html#d4e706)ni, [green open access](https://sparontologies.github.io/pso/current/pso.html#d4e720)ni, [in press](https://sparontologies.github.io/pso/current/pso.html#d4e733)ni, [initial draft](https://sparontologies.github.io/pso/current/pso.html#d4e746)ni, [intermediate draft](https://sparontologies.github.io/pso/current/pso.html#d4e760)ni, [libre open access](https://sparontologies.github.io/pso/current/pso.html#d4e773)ni, [non-confidential](https://sparontologies.github.io/pso/current/pso.html#d4e786)ni, [open access](https://sparontologies.github.io/pso/current/pso.html#d4e800)ni, [peer reviewed](https://sparontologies.github.io/pso/current/pso.html#d4e813)ni, [proof](https://sparontologies.github.io/pso/current/pso.html#d4e828)ni, [published](https://sparontologies.github.io/pso/current/pso.html#d4e844)ni, [rejected for publication](https://sparontologies.github.io/pso/current/pso.html#d4e857)ni, [republished](https://sparontologies.github.io/pso/current/pso.html#d4e870)ni, [restricted access](https://sparontologies.github.io/pso/current/pso.html#d4e884)ni, [retracted from publication](https://sparontologies.github.io/pso/current/pso.html#d4e897)ni, [reviewed](https://sparontologies.github.io/pso/current/pso.html#d4e910)ni, [revised](https://sparontologies.github.io/pso/current/pso.html#d4e926)ni, [submitted](https://sparontologies.github.io/pso/current/pso.html#d4e939)ni, [subscription access](https://sparontologies.github.io/pso/current/pso.html#d4e952)ni, [under review](https://sparontologies.github.io/pso/current/pso.html#d4e966)ni, [unpublished](https://sparontologies.github.io/pso/current/pso.html#d4e979)ni, [version of record](https://sparontologies.github.io/pso/current/pso.html#d4e992)ni, [withdrawn from submission](https://sparontologies.github.io/pso/current/pso.html#d4e1006)ni

Not sure if it is applicable to our use case but for payment they have [ArticleProcessingChargeStatus](http://purl.org/spar/pso/ArticleProcessingChargeStatus) with:
[APC fully waived](https://sparontologies.github.io/pso/current/pso.html#d4e466)ni, [APC not charged](https://sparontologies.github.io/pso/current/pso.html#d4e480)ni, [APC not required](https://sparontologies.github.io/pso/current/pso.html#d4e493)ni, [APC paid](https://sparontologies.github.io/pso/current/pso.html#d4e506)ni, [APC partially waived](https://sparontologies.github.io/pso/current/pso.html#d4e520)ni

**Comment by lzehl** [comment]
MORE REVISIONS (@Raphael-Gazzotti sorry for required changes resulting from this):

Accessibility
- channel: AccessChannel  (1) { physical access | virtual access | hybrid access }
- form: AccessForm (1) { direct access | mediated access }
- eligibility: AccessEligibilityType { open access | controlled access | restricted access }
- process: AccessProcessType (1) { immediate access | registered access (login - no authentication) | authenticated access (login with authentication) | provider-approved access (manual approval by owner or admin)  | committee-approved access (manual approval by some governing committee) }
- paymentModel: PaymentModelType (1–N) { zeroCostPaymentModel | singlePaymentPaymentModel | fixedRecurringPaymentModel | unitBasedPaymentModel | consumptionBasedPaymentModel | transactionBasedPaymentModel | stepPricingPaymentModel | allowanceOveragePaymentModel | takeRatePaymentModel | revenueSplitPaymentModel | retainerPaymentModel | performanceBasedPaymentModel }

publicationStatus should become a direct property of RPVs.

@apdavison @olinux @Raphael-Gazzotti would you agree to this revised version? I tried to get the orthogonality right (so the combinatorics work) and lighten it a bit regarding the access processing types. 

For later (v6.0) to discuss: asserting accessibility to optionally also to lower levels than repo (files and bundles). allowing repo/rpv to state multiple accessibility options (if there is a mix)?

**Comment by apdavison** [comment]
For AccessProcessType, "login" implies authentication. Did you mean "authorization"?

In general, I think the terms suggested on November 12th 2025 make much more sense, except I would remove "immediate" (If there is no process, the value should be `null`)

**Comment by lzehl** [comment]
@apdavison @Raphael-Gazzotti for now I would suggest to do:

process: AccessProcessType (1) { immediate access | registered access (no authentication) | authenticated access | authorized access (authorization required by someone) }

we can then think about an optional property that specifies the authorizationAuthorityLevel (?) { provider | administrator | governance body } ??
